powerquery-docs/connectors/dataverse.md

```diff
@@ -175,7 +175,7 @@ Misspelling a column name might result in an error message about query folding i
 
 ### Accessing large semantic models
 
-Power BI semantic models contained in Dataverse can be very large. If you're using the Power Query Dataverse connector, any specific query that accesses the model has a fixed five (5) minute timeout irrespective of the size of the data. For more information, go to [limitations](/power-apps/developer/data-platform/dataverse-sql-query#limitations). So you might need to query the data multiple times to access all of the data in the model. Using multiple queries can take a considerable amount of time to return all the data.
+Queries on Dataverse tables can return large amounts of data. If you're using the Power Query Dataverse connector, any specific query that accesses the model has a fixed five (5) minute timeout irrespective of the size of the data. For more information, go to [limitations](/power-apps/developer/data-platform/dataverse-sql-query#limitations). So you might need to query the data multiple times to access all of the data in the model. Using multiple queries can take a considerable amount of time to return all the data.
```

Comment:
Since "Power BI semantic models" was changed, should the heading above this paragraph change as well (since it's called "Accessing large semantic models")?